### ISSUE TITLE
Android: CVV of bank card not censored

### ISSUE THREAD

**Issue by julianpoemp** [buttercup/buttercup-mobile#263]
I created a new bank card entry with the new Buttercup 2.0 desktop client. After it was saved I went into the Buttercup Android app and looked for the newly created entry. I found out that the CVV is not censored, instead the card number is censored.

Shouldn't be the CVV (Pin?) censored instead of the card number?

System:
Andorid 11
Device: Samsung Galaxy S21+
App Version: 1.16.0

**Comment by perry-mitchell** [comment]
Yeah, it's probably because the card number is saved as "password" in the guts of the vault.. So it should probably be changed to avoid password and the CVV set to hidden.
